go.mod

```diff
@@ -80,7 +80,7 @@ require (
 	github.com/rudderlabs/rudder-go-kit v0.38.0
 	github.com/rudderlabs/rudder-observability-kit v0.0.3
 	github.com/rudderlabs/rudder-schemas v0.5.1
-	github.com/rudderlabs/rudder-transformer/go v0.0.0-20240812044419-23196ec42acf
+	github.com/rudderlabs/rudder-transformer/go v0.0.0-20240906042448-f7783d8fb300
```

Comment:
Do we need to revert test cases as well ?

Reply:
My bad, we are moving forward

Reply:
these are failing so just upgrade the transformer version